libs/components/forms/src/lib/modules/file-attachment/file-attachment/file-attachment.component.spec.ts

```diff
@@ -576,6 +576,7 @@ describe('File attachment', () => {
     ];
 
     setupStandardFileChangeEvent(file);
+    await fixture.whenStable();
```

Comment:
These are needed now that the file handling is truly asynchronous. Before, we were mocking out `FileReader` directly, making it a synchronous action, but that's a bit deceptive. The mocks are now more accurate to the actual inner workings of the component.

Reply:
should we update the mocks to use the file reader testing service? im also ok with leaving it like this if it's a big change

Reply:
from chat: using the testing service with file attachment tests would get messy right now so will be done during the file attachment refactor